packages/stage-ui/src/styles/scroll-fade.css

```
@@ -0,0 +1,520 @@
+@property --scroll-fade-e {
+  syntax: "<length-percentage>";
+  inherits: false;
+  initial-value: 0px;
+}
+
+@property --scroll-fade-mask {
+  syntax: "*";
+  inherits: false;
+}
+
+@theme inline {
```

Stylelint (17.14.0)

[error] Unexpected unknown at-rule "`@theme`" (scss/at-rule-no-unknown)

(scss/at-rule-no-unknown)